libkbfs/block_types.go

```diff
@@ -99,6 +99,8 @@ type FileBlock struct {
 	Contents []byte `codec:"c,omitempty"`
 	// if indirect, contains the indirect pointers to the next level of blocks
 	IPtrs []IndirectFilePtr `codec:"i,omitempty"`
+
+	hash *RawDefaultHash
```

Comment:
Please add a comment above this explaining that this is a hash of the _plaintext_ contents of the block, and is only set on direct file blocks.